examples/next/src/pages/player.tsx

```diff
@@ -0,0 +1,52 @@
+import { Player } from '@livepeer/react';
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useMemo } from 'react';
+
+const isTrue = (b: string) =>
+  b === '' || b === '1' || b?.toLowerCase() === 'true';
+
+function toStringValues(obj: Record<string, any>) {
+  const strObj: Record<string, string> = {};
+  for (const [key, value] of Object.entries(obj)) {
+    strObj[key] = value.toString();
+  }
+  return strObj;
+}
+
+const PlayerPage: NextPage = () => {
```

Comment:
Is this reusable for the lvpr.tv repo somehow? Seems like it's the exact same code right 👀